ecosystem/logproxy/logproxy.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package logproxy forwards Trillian Log Server requests to another server.
+package logproxy
+
+import (
+	context "golang.org/x/net/context"
+
+	google_protobuf "github.com/golang/protobuf/ptypes/empty"
+	"github.com/google/trillian"
+)
+
+// Proxy forwards requests to the given TrillianLogClient
```

Comment:
This type will stutter (`logproxy.Proxy`), so it would be nice to tweak the naming somehow; also, we should plan ahead for something similar being created for Maps.  So maybe `proxy.Log`, with the possibility of `proxy.Map` in the future?